src/FamilyHubs.ReferralUi.Ui/Services/Api/OpenReferralOrganisationClientService.cs

```diff
@@ -61,7 +62,7 @@ public async Task<List<OrganisationDto>> GetListOrganisations()
         var request = new HttpRequestMessage
```

Comment:
we need to add new test to cover this method, the current tests are not testing this method at all

Reply:
It now has a unit test.